aws-py-static-website/__main__.py

```diff
@@ -0,0 +1,166 @@
+import json
+import mimetypes
+import os
+
+from pulumi import export, FileAsset, ResourceOptions, Config, Output
+import pulumi_aws
+import pulumi_aws.acm
+import pulumi_aws.cloudfront
+import pulumi_aws.config
+import pulumi_aws.route53
+import pulumi_aws.s3
+
+def get_domain_and_subdomain(domain):
+    parts = domain.split('.')
+    if len(parts) < 2:
+        raise Exception(f'No TLD found on ${domain}')
+    if len(parts) == 2:
+        return "", domain
+    subdomain = parts[0]
+    parts.pop(0)
+    return subdomain, '.'.join(parts) + '.'
+
+stack_config = Config('static-website')
+target_domain = stack_config.require('targetDomain')
+path_to_website_contents = stack_config.require('pathToWebsiteContents')
+certificate_arn = stack_config.get('certificateArn')
+
+content_bucket = pulumi_aws.s3.Bucket('contentBucket',
+    bucket=target_domain,
+    acl='public-read',
+    website={
+        'index_document': 'index.html',
+        'error_document': '404.html'
+    })
+
+def crawl_directory(content_dir, f):
```

Comment:
I'd strongly suggest putting a doc comment on this function, since it isn't immediately obvious what `f` is used for.